src/SuperDumpService/Services/Analyzers/AnalyzerJob.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using SuperDumpService.Models;
+
+namespace SuperDumpService.Services.Analyzers {
+	public enum AnalyzerState {
+		/// <summary>
+		/// No Further Analyzers should be executed.
+		/// </summary>
+		Cancel,
```

Comment:
nit: shouldn't the wording rather be like "Cancelled", "Succeeded", "Failed", "Initialized"?

Reply:
You are right. I changed it.